meta-mender-core/classes/mender-bootstrap-artifact.bbclass

```diff
@@ -66,6 +66,21 @@ IMAGE_CMD:bootstrap-artifact() {
             bberror "The image checksum cannot be empty"
         fi
 
+        modules_provides=""
```

Comment:
Q: Why don't share this functionality?

Reply:
Good question (it was also suggested in the acceptance criteria to share the functionality).

But @oleorhagen why is not already shared everything else for the Artifact parameters between `mender-artifactimg.bbclass` and `mender-bootstrap-artifact.bbclass`? That is what put me off to share this new part.

Reply:
Because they don't take all the same parameters. No extra arguments allowed for the `bootstrap` Artifact

Reply:
Right. But that is the only exception. Everything else is applied the same way.

But okey, I'll refactor my part :+1: